lib/spack/spack/solver/asp.py

```diff
@@ -2222,99 +2180,11 @@ def external_packages(self):
             if pkg_name not in self.pkgs:
                 continue
 
-            # Check if the external package is buildable. If it is
-            # not then "external(<pkg>)" is a fact, unless we can
-            # reuse an already installed spec.
             external_buildable = data.get("buildable", True)
-            externals = data.get("externals", [])
-            if not external_buildable or externals:
-                self.gen.h2(f"External package: {pkg_name}")
-
             if not external_buildable:
```

Comment:
Can be inlined at this point:

```suggestion
            if not data.get("buildable", True):
```